components/loginform.js

```diff
@@ -17,29 +14,24 @@ class LoginForm extends Component {
     }
   }
 
-  setLocalState = (state) => {
```

Comment:
why get rid of this? when a user changes state (starts typing) don't we want to remove the error?

Reply:
I don't think so. I just failed logging into Twitter and the message is left up until I hit the button again (regardless of typing).

Reply:
we need a ux. w/e fine for now